evm/utils/db.py

```diff
@@ -30,3 +41,20 @@ def get_block_header_by_hash(block_hash, db):
     Returns the header for the parent block.
     """
     return db.get_block_header_by_hash(block_hash)
+
+
+def get_empty_root_hash(db):
```

Comment:
This strikes me as a good `classmethod` for the `ChainDB` class.  That's basically what it currently is doing as I believe that you could do the following and it would just work:

```python
class ChainDB:
    get_empty_root_hash = classmethod(get_empty_root_hash)
```

I'm not proposing we do the above, but more stating that the current implementation is equivalent to just implementing this as a class method.